index.js

```diff
@@ -217,15 +230,19 @@ function adapter(uri, opts){
 
     debug('removing %s from rooms %s', id, Object.keys(rooms));
 
-    Adapter.prototype.delAll.call(this, id);
```

Comment:
I disagree with socket.io-redis's implementation. Most of the logic they have is already in the socket-adapter base class and not necessary. Could you explain to me the situation you're having issues with?